src/input/mod.rs

```diff
@@ -942,7 +941,18 @@ impl State {
                                     }
                                 };
                                 if !done {
-                                    if let Some((target, _)) =
+                                    if let Some(or) =
+                                        self.common.shell.override_redirect_windows.iter().find(
+                                            |or| {
+                                                or.is_in_input_region(
+                                                    &(pos.as_logical()
+                                                        - or.geometry().loc.to_f64()),
+                                                )
+                                            },
+                                        )
```

Comment:
This seems to be the same as the code that was changed later in this file (in e8a9a373e82a4d35e8e611c79b741bae7c4ac1be). Presumably the same fix is needed here?

Maybe the redundancy could also be improved somehow.

Reply:
Actually it's not a &&X11Surface here.

Reply:
fixed. good catch